nucypher/blockchain/eth/sol/source/contracts/StakingEscrow.sol

```diff
@@ -1300,24 +1294,24 @@ contract StakingEscrow is Issuer, IERC900History {
         internal
     {
         // Check that the old sub stake should be saved
-        bool oldCurrentConfirmedPeriod = _info.currentConfirmedPeriod != 0 &&
-            _info.currentConfirmedPeriod < _currentPeriod;
-        bool oldNextConfirmedPeriod = _info.nextConfirmedPeriod != 0 &&
-            _info.nextConfirmedPeriod < _currentPeriod;
-        bool crossCurrentConfirmedPeriod = oldCurrentConfirmedPeriod && _info.currentConfirmedPeriod >= _firstPeriod;
-        bool crossNextConfirmedPeriod = oldNextConfirmedPeriod && _info.nextConfirmedPeriod >= _firstPeriod;
-        if (!crossCurrentConfirmedPeriod && !crossNextConfirmedPeriod) {
+        bool oldcurrentCommittedPeriod = _info.currentCommittedPeriod != 0 &&
```

Comment:
Any reason for the lack of camel case for `oldcurrent`? Not a big deal though, just curious.

Reply:
my bad, fixed, thanks
✔️